site/content/entry/yuzu-progress-report-jul-2021/index.md

```diff
@@ -0,0 +1,482 @@
+
+{{< single-title-imgs
+    "Well excuse me, Princess (The Legend of Zelda: Breath of the Wild)"
+    "./botwbug.png"
+    "./botwfix.png"
+    >}}
+
+[K0bin](https://github.com/K0bin) arrived to give us a hand, fixing an important screw up we made.
+
+Prior to the introduction of full support on Resizable BAR in modern GPUs and systems, the PCI-Express standard is limited to a buffer of 256MB in video memory to communicate 
+to the CPU at a time.
+yuzu uses this small portion of VRAM for allocating its staging buffer, but if the user has GPU intensive background applications, there may not be enough space to allow the 
+allocation to happen, and yuzu would refuse to create a Vulkan instance, failing to boot any game.
+OpenGL is, as usual, excluded from this issue thanks to letting the GPU driver handle those allocations on its own.
+
+K0bin fixes this issue by [doing the allocation in system RAM](https://github.com/yuzu-emu/yuzu/pull/6740) if there isn’t enough free space. Many thanks!
+
+## Stop This Sound!
+
+[Maide](https://github.com/Kelebek1) has been dedicated to improving the audio side of yuzu this month.
+By [introducing some missing PCM formats](https://github.com/yuzu-emu/yuzu/pull/6564), the missing audio present in `Ys IX: Monstrum Nox` was finally fixed.
+
+`PCM` stands for [Pulse-Code Modulation](https://en.wikipedia.org/wiki/Pulse-code_modulation), an encoding technique used to represent analogue audio signals digitally.
+These signals have two basic properties: The [sample rate](https://en.wikipedia.org/wiki/Sampling_(signal_processing)#Sampling_rate) (how many samples of the signal are taken 
+per second), and the [bit-depth](https://en.wikipedia.org/wiki/Audio_bit_depth) (how many bits are used to represent the "loudness" of the signal's samples at any given point 
+in time).
+
+In this PR, Maide introduced a number of methods to process formats that were missing in the current implementation — namely, the ability to decode PCM files encoded with a 
+bit-depth of 8-bit, 32-bit, and also floating-point values.
+Since previously none of these formats were being decoded by yuzu, any audio file that made use of them was not being reproduced, causing this behaviour.
+
+`Tales of Vesperia` was another title with sound problems, in which audio would be played monaurally through the left channel — far from a pleasant experience, as you can hear 
+here:
+
+{{< audio "./vesperiabug.mp3" >}}
+
+This game in particular would request the available number of active channels and cap its output based on this information — in other words, the game would not output audio to 
+more channels than what yuzu reported.
+Since yuzu was always returning a value of `1`, the game ended up outputting all the audio into the left channel.
+This problem was thus fixed by [reporting two channels as active instead of one](https://github.com/yuzu-emu/yuzu/pull/6567), which is now mixed properly:
```

Comment:
```suggestion
Thus, this problem was fixed by [reporting two channels as active instead of one](https://github.com/yuzu-emu/yuzu/pull/6567), which is now mixed properly:
```